src/libs/focusComposerWithDelay.ts

```diff
@@ -20,13 +25,27 @@ function focusComposerWithDelay(textInput: TextInput | null): FocusComposerWithD
 
         if (!shouldDelay) {
             textInput.focus();
+            if (forceSetSelection) {
+                if (Platform.OS === 'web') {
+                    (textInput as HTMLTextAreaElement).setSelectionRange(forceSetSelection.start, forceSetSelection.end);
+                } else {
+                    (textInput as TextInput).setSelection(forceSetSelection.start, forceSetSelection.end);
+                }
+            }
             return;
         }
         ComposerFocusManager.isReadyToFocus().then(() => {
             if (!textInput) {
                 return;
             }
             textInput.focus();
+            if (forceSetSelection) {
+                if (Platform.OS === 'web') {
```

Comment:
```suggestion
                if (Platform.OS === CONST.PLATFORM.WEB) {
```

Reply:
Thanks, addressed in 7fc489c77a22ea300367d5c295ee396de5c24668